daemons/based/pacemaker-based.c

```diff
@@ -52,147 +55,178 @@ static bool startCib(const char *filename);
 extern int write_cib_contents(gpointer p);
 void cib_cleanup(void);
 
+static crm_exit_t exit_code = CRM_EX_OK;
+
 static void
 cib_enable_writes(int nsig)
 {
     crm_info("(Re)enabling disk writes");
     cib_writes_enabled = TRUE;
 }
 
-static pcmk__cli_option_t long_options[] = {
-    // long option, argument type, storage, short option, description, flags
-    {
-        "help", no_argument, 0, '?',
-        "\tThis text", pcmk__option_default
-    },
-    {
-        "verbose", no_argument, NULL, 'V',
-        "\tIncrease debug output", pcmk__option_default
-    },
-    {
-        "stand-alone", no_argument, NULL, 's',
-        "\tAdvanced use only", pcmk__option_default
-    },
-    {
-        "disk-writes", no_argument, NULL, 'w',
-        "\tAdvanced use only", pcmk__option_default
-    },
-    {
-        "cib-root", required_argument, NULL, 'r',
-        "\tAdvanced use only", pcmk__option_default
-    },
-    { 0, 0, 0, 0 }
+/*!
+ * \internal
+ * \brief Set up options, users, and groups for stand-alone mode
+ *
+ * \param[out] error  GLib error object
+ *
+ * \return Standard Pacemaker return code
+ */
+static int
+setup_stand_alone(GError **error)
+{
+    int rc = 0;
+    struct passwd *pwentry = NULL;
+
+    preserve_status = TRUE;
+    cib_writes_enabled = FALSE;
+
+    errno = 0;
```

Comment:
Probably not necessary but the `getpwnam()` man page says to do this.

Reply:
Necessary b/c getpwnam() can return NULL (for entry not found) without setting errno

Reply:
My thought was that it's probably already 0